lib/danger/request_sources/bitbucket_cloud.rb

```diff
@@ -70,15 +70,14 @@ def organisation
       def update_pull_request!(warnings: [], errors: [], messages: [], markdowns: [], danger_id: "danger", new_comment: false, remove_previous_comments: false)
         delete_old_comments(danger_id: danger_id) if !new_comment || remove_previous_comments
 
-        comment = generate_description(warnings: warnings, errors: errors, template: "bitbucket_server")
-        comment += "\n\n"
+        generate_description(warnings: warnings, errors: errors, template: "bitbucket_server")
```

Comment:
This deletion feels more like exposing a bug - shouldn't we be updating the markdown for the main comment with this variable?

Reply:
Good point. I reviewed again and I thought #1428 forgot to remove it. So I removed it, and added `template: "bitbucket_server"` to L80. WDYT?